tests/unit/LocalDB.js

```diff
@@ -253,9 +253,10 @@ define([
 					]);
 				});
 			},
-			"in('mappedTo', letterStore.filter({vowel: true}).select('id'))":
+			"in('mappedTo', letterStore.filter({vowel: true}).select('id'))": function() {
```

Comment:
I'm a little confused as to why this `testQuery` call was wrapped in a function, given that `testQuery` already returns a function which returns a promise.  Wouldn't this bork reporting of the results of this async test?